libethereum/Block.h

```diff
@@ -191,16 +191,16 @@ class Block
 	h256Hash const& pendingHashes() const { return m_transactionSet; }
 
 	/// Get the transaction receipt for the transaction of the given index.
-	TransactionReceipt const& receipt(unsigned _i) const { return m_receipts[_i]; }
+	TransactionReceipt const& receipt(unsigned _i) const { return m_receipts.at(_i); }
```

Comment:
Is there a catch for `std::out_of_range` or at least `std::exception` everywhere where it's called?